gin-client/repos.go

```diff
@@ -250,15 +269,33 @@ func (gincl *Client) CloneRepo(repoPath string) (string, error) {
 		return "", err
 	}
 
+	err = AddRemote("origin", remotePath)
+	// Ignore if it already exists
+	if err != nil && !strings.Contains(err.Error(), "already exists") {
+		return "", initerr
+	}
+
+	if new {
```

Comment:
Could this `if new` be combined with the `if new` at L290 or should they remain separate?